cob_hardware_config/cob3-2/config/robot_modules.yaml

```diff
@@ -0,0 +1 @@
+robot_modules: ['base','tray','torso','head','arm','sdh']
```

Comment:
what did you find out about the ```robot_modules.yaml``` file?
Is it needed in cob_hardware_config or cob_default_robot_config? Or both?